pkg/controller/servicebindingrequest/retriever.go

```diff
@@ -197,7 +197,7 @@ func (r *Retriever) readSecret(cr *unstructured.Unstructured, name string, items
 		// update cache after reading configmap/secret in cache
 		r.cache[fromPath].(map[string]interface{})[path].(map[string]interface{})[k] = string(data)
 		// making sure key name has a secret reference
-		r.store(cr, fmt.Sprintf("secret_%s", k), data)
+		r.store(envVarPrefix, cr, fmt.Sprintf("secret_%s", k), data)
```

Comment:
May I request an associated change? 
If the user has specified an `envVarPrefix : ""`, could we even skip the `secret_xyz` / `configmap_xyz` ?